src/Raven.Studio/wwwroot/App/views/database/settings/databaseSettings.html

```diff
@@ -0,0 +1,231 @@
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-7 add-vertical-split">
+                    <div class="form-group no-margin">
+                        <div class="margin-top margin-top-sm">
+                            <small data-placement="left" data-toggle="tooltip" data-animation="true"
+                                   data-bind="attr: { title: hasServerValue() ? 'This is a server-wide configuration' : 'This is the default configuration' }">
+                            <!-- Waiting for icons from RavenDB-14970 -->
+                            <!-- <i class="source-icon" data-bind="css: { 'icon-xxx': hasServerValue(), 'source-icon-null': !serverOrDefaultValue() }"></i> -->
+                            <!-- <i class="source-icon" data-bind="css: { 'icon-yyy': !hasServerValue(), 'source-icon-null': !serverOrDefaultValue() }"></i> -->
+                                <span data-bind="text: serverOrDefaultValue() || 'null'" id="value"></span>
+                            </small>
+                            <small data-placement="right" data-toggle="tooltip" data-animation="true" data-html="true" data-bind="visible: isServerWideOnlyEntry"
+                                   title="This is a server-wide entry.<br>It can be edited only from the <strong>settings.json</strong> file located in your RavenDB executable folder.">
+                                <small class="margin-left margin-left-sm"><i class="icon-lock"></i></small>
+                            </small>
+                        </div>
+                    </div>
+                    <div data-bind="template: { name: isServerWideOnlyEntry() ? 'db-settings-serverWide-template' : 'db-settings-override-template' }"></div>
+                </div>
+            </div>
+        </div>
+    </form>
+</script>
+
+<script type="text/html" id="db-settings-serverWide-template">
+    <div class="margin-top margin-bottom"></div>
+</script>
+
+<script type="text/html" id="db-settings-override-template">
+    <div class="form-group no-margin margin-left margin-bottom margin-bottom-sm padding padding-xs no-padding-right">
+        <div class="toggle">
+            <input type="checkbox" data-bind="checked: override, attr: {'id': 'overrideToggle_' + keyName() }" />
+            <label title="Customize the database value" data-bind="attr: { 'for': 'overrideToggle_' + keyName() }">
+                <span>Override</span>
+            </label>
+        </div>
+        <div data-bind="collapse: override" class="margin-top margin-top-lg override-options flex-grow">
+            <div class="margin-top margin-top-sm"  data-bind="collapse: override, validationElement: customizedDatabaseValue">
+                <div data-bind="template: { name: 'db-settings-' + getTemplateType() + '-template' }"></div>
+            </div>
+        </div>
+    </div>
+</script>
+
+<!-- String -->
```

Comment:
please remove those comments - you duplicate info from next line

Reply:
done